uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestStatusMatrixTest.php

```diff
@@ -185,4 +258,18 @@ public function allNonPossibleStatusTransitionForLeaveApprover() {
       ['', $leaveRequestStatuses['cancelled']],
     ];
   }
+
+  /**
+   * Return possible status transitions that is only exclusive to the Manager or Admin
+   *
+   * @return array
+   */
+  public function getManagerExclusivePossibleStatusTransitions() {
```

Comment:
Some of the tests in this class actually use data providers. The reason why I did not add it for the newer tests is because, some of them use contact fabricators and these contacts have to be fabricated each time the data provider supplies values to the test, this caused the tests to run slower.